### ISSUE TITLE
Representing different parts of a Standard and Different Versions of a standard

### ISSUE THREAD

**Issue by PriyankaNanjappa** [i40-Tools/StandardOntology#38]
As of now, We do not have some property representing the sub-parts  of a standard.
For example, [This instance of a standard](https://webstore.iec.ch/publication/1172) is talking about part 27 of the standard IEC 60255. We need to represent this meaningfully in the schema.

We have the property sto:hasEdition to deal with different versions of the same standard.

**Comment by igrangel** [comment]
Yes, we have the property [PartOf](http://ontologydesignpatterns.org/wiki/Submissions:PartOf), which is one [Ontology Design Pattern](http://ontologydesignpatterns.org/wiki/Community:ListPatterns) (ODP from now one). But, after, checking this deeper I think the [Componency](http://ontologydesignpatterns.org/wiki/Submissions:Componency) ODP may be better to represent this kind of relations since it is not-transitive and the PartOf is transitive.

**Comment by PriyankaNanjappa** [comment]
When we talk about  IEC 60255 and it's parts or any other standard and it's parts, it's worth noting that there is no generic Document called IEC 60255. There exists only documents that talk about it's parts. Starting with IEC 60255-1:2009 . Hence, Componency and PartOf might not be appropriate since we might fail to create the top level instance IEC 60255 since there is no document that refers to this in real world.

**Comment by PriyankaNanjappa** [comment]
I see that the property [isComponentOf](https://github.com/i40-Tools/StandardOntology/blob/semantic_lab/sto.ttl#L81) is created for a standard. Do we need top implement this to all other standards?

**Comment by igrangel** [comment]
Whenever makes sense, yes, i.e., that a given Standard is a component of other like in the AutomationML case.

**Comment by PriyankaNanjappa** [comment]
Could you please explain the role of isComponentOf relation in [this](https://github.com/i40-Tools/StandardOntology/blob/c6f110dd12cdbf96f46a6c207f395537df9165c6/sto.ttl#L5104) part of the code
